logstash-core/spec/api/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 # encoding: utf-8
 API_ROOT = File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "lib", "logstash", "api"))
 
+require "stud/task"
```

Comment:
because if you try to run only the api test with `bin/rspec` without this in the helper it does not work.